### ISSUE TITLE
SimpleBitfinexApiBroker - OnError called java.lang.IllegalArgumentException: Unable to find order type for: INSUFFICIENT BALANCE (U1) was: PARTIALLY FILLED

### ISSUE THREAD

**Issue by GeekHare** [jnidzwetzki/bitfinex-v2-wss-api-java#308]
```
[ERROR] 2023-04-17 08:42:40.058 [ tyrus-jdk-client-1000] SimpleBitfinexApiBroker - OnError called java.lang.IllegalArgumentException: Unable to find order type for: INSUFFICIENT BALANCE (U1) was: PARTIALLY FILLED @ 98.423(-0.10137227)
	at com.github.jnidzwetzki.bitfinex.v2.entity.BitfinexSubmittedOrderStatus.fromString(BitfinexSubmittedOrderStatus.java:50)
	at com.github.jnidzwetzki.bitfinex.v2.callback.channel.account.info.OrderHandler.jsonToBitfinexSubmittedOrder(OrderHandler.java:118)
	at com.github.jnidzwetzki.bitfinex.v2.callback.channel.account.info.OrderHandler.handleChannelData(OrderHandler.java:74)
	at com.github.jnidzwetzki.bitfinex.v2.callback.channel.AccountInfoHandler.handleChannelData(AccountInfoHandler.java:129)
	at com.github.jnidzwetzki.bitfinex.v2.SimpleBitfinexApiBroker.handleChannelCallback(SimpleBitfinexApiBroker.java:608)
	at com.github.jnidzwetzki.bitfinex.v2.SimpleBitfinexApiBroker.websocketCallback(SimpleBitfinexApiBroker.java:536)
	at com.github.jnidzwetzki.bitfinex.v2.WebsocketClientEndpoint.onMessage(WebsocketClientEndpoint.java:119)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.glassfish.tyrus.core.AnnotatedEndpoint.callMethod(AnnotatedEndpoint.java:556)
	at org.glassfish.tyrus.core.AnnotatedEndpoint$WholeHandler$1.onMessage(AnnotatedEndpoint.java:655)
	at org.glassfish.tyrus.core.TyrusSession.notifyMessageHandlers(TyrusSession.java:591)
	at org.glassfish.tyrus.core.TyrusEndpointWrapper.onMessage(TyrusEndpointWrapper.java:861)
	at org.glassfish.tyrus.core.TyrusWebSocket.onMessage(TyrusWebSocket.java:196)
	at org.glassfish.tyrus.core.frame.TextFrame.respond(TextFrame.java:116)
	at org.glassfish.tyrus.core.ProtocolHandler.process(ProtocolHandler.java:819)
	at org.glassfish.tyrus.client.TyrusClientEngine$TyrusReadHandler.handle(TyrusClientEngine.java:726)
	at org.glassfish.tyrus.container.jdk.client.ClientFilter.processRead(ClientFilter.java:204)
	at org.glassfish.tyrus.container.jdk.client.Filter.onRead(Filter.java:111)
	at org.glassfish.tyrus.container.jdk.client.Filter.onRead(Filter.java:113)
	at org.glassfish.tyrus.container.jdk.client.SslFilter.handleRead(SslFilter.java:402)
	at org.glassfish.tyrus.container.jdk.client.SslFilter.processRead(SslFilter.java:365)
	at org.glassfish.tyrus.container.jdk.client.Filter.onRead(Filter.java:111)
	at org.glassfish.tyrus.container.jdk.client.Filter.onRead(Filter.java:113)
	at org.glassfish.tyrus.container.jdk.client.TransportFilter$4.completed(TransportFilter.java:295)
	at org.glassfish.tyrus.container.jdk.client.TransportFilter$4.completed(TransportFilter.java:279)
	at java.base/sun.nio.ch.Invoker.invokeUnchecked(Invoker.java:129)
	at java.base/sun.nio.ch.Invoker.invokeDirect(Invoker.java:160)
	at java.base/sun.nio.ch.UnixAsynchronousSocketChannelImpl.implRead(UnixAsynchronousSocketChannelImpl.java:573)
	at java.base/sun.nio.ch.AsynchronousSocketChannelImpl.read(AsynchronousSocketChannelImpl.java:277)
	at java.base/sun.nio.ch.AsynchronousSocketChannelImpl.read(AsynchronousSocketChannelImpl.java:298)
	at java.base/java.nio.channels.AsynchronousSocketChannel.read(AsynchronousSocketChannel.java:425)
	at org.glassfish.tyrus.container.jdk.client.TransportFilter._read(TransportFilter.java:279)
	at org.glassfish.tyrus.container.jdk.client.TransportFilter$4.completed(TransportFilter.java:297)
	at org.glassfish.tyrus.container.jdk.client.TransportFilter$4.completed(TransportFilter.java:279)
	at java.base/sun.nio.ch.Invoker.invokeUnchecked(Invoker.java:129)
	at java.base/sun.nio.ch.Invoker$2.run(Invoker.java:221)
	at java.base/sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:113)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

**Comment by GeekHare** [comment]
@jnidzwetzki, I've fixed it and created a PR #309

**Comment by jnidzwetzki** [comment]
@GeekHare Thanks for the contribution. The PR looks good!
